views/admin-page-status-manager.php

```diff
@@ -59,7 +59,7 @@
                         <select name="status_alias">
                             <option value=""><?php _e( '— Select —', 'trackmage' ); ?></option>
                             <?php foreach ( $aliases as $id => $name ) : ?>
-                                <option value="<?php echo $id; ?>"><?php echo $name; ?></option>
+                                <option value="<?php echo $id; ?>" <?php echo isset($used_aliases[$id])?'style="display: none"':'';?>><?php echo $name; ?></option>
```

Comment:
The status manager is powered by javascript. Will options change dynamically or page refresh is always needed?
How about backend validation?

Reply:
These changes are related only for receiving statuses from TM. I'm working on task 

> The status manager is powered by javascript. Will options change dynamically or page refresh is always needed?
> How about backend validation?

These changes are related only for receiving order statuses from TM. I'm still working on hiding used statuses. 
What do you want to validate in backend? I though backend validation will be used only on edit/add status.

Reply:
>These changes are related only for receiving order statuses from TM

This hides the status option, doesn't it? `style="display: none"`

>I though backend validation will be used only on edit/add status.

True.

Reply:
> > These changes are related only for receiving order statuses from TM
> 
> This hides the status option, doesn't it? `style="display: none"`
> 
> > I though backend validation will be used only on edit/add status.
> 
> True.

Yep, but it's code to hide status directly after page opens. I'm working on JS code to hide/show TM statuses on edit/add status

Reply:
:+1: okay